### PR TITLE
Updated the empty config state

### PR DESCRIPTION
**Pull request by tanya-rai**
Updated the empty config state

Made it cleaner - this should be intuitive to use as is.

Also user doesn't have to delete as much text from initial state to jump in and get started.

### CODE REVIEW

**Comment by tanya-rai** [comment]
Screenshot 2024-02-21 at 8 10 43 PM

File: vscode-extension/static/untitled.aiconfig.json [hunk 1]
```diff
@@ -11,21 +12,21 @@
     },
     "default_model": "gpt-3.5-turbo"
   },
-  "description": "Fill out a description about your AIConfig here.",
+  "description": "Enter a description for this AIConfig",
```
Comment (rossdanlm) [inline vscode-extension/static/untitled.aiconfig.json:15]: I think a good compromise between keeping it clean and informative would be to link it to our AICOnfig getting started docs

File: vscode-extension/static/untitled.aiconfig.json [hunk 2]
```diff
@@ -11,21 +12,21 @@
     },
     "default_model": "gpt-3.5-turbo"
   },
-  "description": "Fill out a description about your AIConfig here.",
+  "description": "Enter a description for this AIConfig. Learn about features in our [docs](https://aiconfig.lastmileai.dev/docs/category/overview)!",
```
Comment (rossdanlm) [inline vscode-extension/static/untitled.aiconfig.json:15]: "Enter a description for this AIConfig. You can also check out [our docs](https://aiconfig.lastmileai.dev/docs/category/overview) to learn more!

**Review by rossdanlm — APPROVED** [review]

**Comment by saqadri** [comment]
It would be really cool to also have sample output saved in the config. Maybe we should have a separate "Create from template" command for that, but now the Untitled is truly just an untitled. My goal was to have an educational untitled because this is the first time someone is seeing an aiconfig

**Pull request merged**